pkg/parser/epub/testdata/package/accessibility-epub3.opf

```diff
@@ -24,6 +24,9 @@
     <link rel="a11y:certifierReport" refines="#certifier2" href="https://example.com/fakereport"/>
     <link rel="a11y:certifierReport" refines="#certifier" href="https://example.com/a11y-report/"/>
 
+    <meta property="a11y:exemption">eaa-microenterprise</meta>
+    <meta property="a11y:exemption">eaa-fundamental-alteration</meta>
+
```

Comment:
Why this testdata package does not have the  `<meta property="a11y:exemption">eaa-disproportionate-burden</meta>`?

Reply:
I've added it